test/producer-consumer-livelock/Makefile

```
@@ -0,0 +1,33 @@
+FILE = producer-consumer
+
+CC = gcc
+
+CPPFLAGS += -I../..
+CFLAGS += -g3 -O0
+LDLIBS += -lpthread
+
+run: $(FILE)
```

checkmake (0.3.2)

[warning] Target "run" should be declared PHONY.

(phonydeclared)